conda-store-server/conda_store_server/utils.py

```diff
@@ -15,6 +15,10 @@ def message(self):
         return self.args[0]
 
 
+class BuildPathError(CondaStoreError):
```

Comment:
I'm more used to having exceptions in an `exceptions.py` module, but I guess there's a precedence to have them in `utils.py`? Probably worth its own issue, not blocking here.

Reply:
Yeah, agree. But not changing it now. Added it next to the existing exception.

Reply:
Let's add an issue to cover this later.